docs/optimization_guide/model_optimization_guide.md

```diff
@@ -18,7 +18,10 @@ Model optimization is an optional offline step of improving the final model perf
 
 .. note:: OpenVINO also supports optimized models (for example, quantized) from source frameworks such as PyTorch, TensorFlow, and ONNX (in Q/DQ; Quantize/DeQuantize format). No special steps are required in this case and optimized models can be converted to the OpenVINO Intermediate Representation format (IR) right away.
 
-Post-training Quantization is the fastest way to optimize a model and should be applied first, but it is limited in terms of achievable accuracy-performance trade-off. In case of poor accuracy or performance after Post-training Quantization, Training-time Optimization can be used as an option.
+Post-training Quantization is the fastest way to optimize a model and should be applied first, but it is limited in terms of achievable accuracy-performance trade-off.
+Therefore, it is recommended to use ``ov.Model()`` as input for ``nncf.quantize()`` method to obtain a quantized model for inference in OpenVINO (see the diagram below). Using a model in the OpenVINO format excludes the additional step of converting an optimized model. It means there is fewer steps in the custom training pipeline and that gives more robust results, especially when there is a need to test and measure the performance of an FP32 precision model. Moreover, the native model representation, that includes all the general optimizations made during the conversion step, may result in better accuracy and performance.
+
+However, in case of unsatisfactory accuracy or performance after Post-training Quantization, Training-time Optimization can be used as an option.
```

Comment:
Probably, such wording will be useful:

Post-training Quantization is the fastest way to optimize a model and should be applied first, but it is limited in terms of achievable accuracy-performance trade-off. The recommend steps to obtain OpenVINO quantized model is to convert the framework model into OpenVINO model and ensure that the model works correctly in OpenVINO, for example through calculating the model metrics. Then feed OpenVINO model to ``nncf.quantize()` method to get the quantized model.

In case of unsatisfactory accuracy or performance after Post-training Quantization, Training-time Optimization can be used as an option.

cc` @MaximProshin

Reply:
I have updated the article with the https://github.com/openvinotoolkit/openvino/pull/19040/commits/fa5a4bd792525553897c40b6f7b2758586b54be5 commit.